rubix_admin/installer.py

```diff
@@ -50,7 +50,15 @@ def _scp(cls, args):
 
     @classmethod
     def _rpm_install(cls, args):
-        return sudo('rpm -U %s %s' %
-                    (args.rpm_args,
-                     os.path.join(args.config["remote_packages_path"],
-                                  os.path.basename(args.rpm))))
+        sudo('rpm -U %s %s' %
+             (args.rpm_args,
+              os.path.join(args.config["remote_packages_path"],
+                           os.path.basename(args.rpm))))
+
+        sudo("cp -a /usr/lib/rubix/lib/* /usr/lib/presto/lib/plugin/hive-hadoop2/")
+        sudo("cp -a /usr/lib/rubix/lib/* /usr/lib/hadoop/lib/")
+        sudo("mkdir -p /media/ephemeral0")
```

Comment:
EMR nodes come with 10gb default space on root partition. Can be changed during the cluster creation though. 

Mount o/p:
proc on /proc type proc (rw,relatime)
sysfs on /sys type sysfs (rw,relatime)
devtmpfs on /dev type devtmpfs (rw,relatime,size=7672784k,nr_inodes=1918196,mode=755)
devpts on /dev/pts type devpts (rw,relatime,gid=5,mode=620,ptmxmode=000)
tmpfs on /dev/shm type tmpfs (rw,relatime)
/dev/xvda1 on / type ext4 (rw,noatime,data=ordered)
none on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,relatime)
/dev/xvdb1 on /emr type xfs (rw,relatime,attr2,inode64,noquota)
/dev/xvdb2 on /mnt type xfs (rw,relatime,attr2,inode64,noquota)
/dev/xvdc on /mnt1 type xfs (rw,relatime,attr2,inode64,noquota)
/dev/xvdd on /mnt2 type xfs (rw,relatime,attr2,inode64,noquota)
/dev/xvde on /mnt3 type xfs (rw,relatime,attr2,inode64,noquota)
cgroup on /cgroup/blkio type cgroup (rw,relatime,blkio)
cgroup on /cgroup/cpu type cgroup (rw,relatime,cpu)
cgroup on /cgroup/cpuacct type cgroup (rw,relatime,cpuacct)
cgroup on /cgroup/cpuset type cgroup (rw,relatime,cpuset)
cgroup on /cgroup/devices type cgroup (rw,relatime,devices)
cgroup on /cgroup/freezer type cgroup (rw,relatime,freezer)
cgroup on /cgroup/hugetlb type cgroup (rw,relatime,hugetlb)
cgroup on /cgroup/memory type cgroup (rw,relatime,memory)
cgroup on /cgroup/perf_event type cgroup (rw,relatime,perf_event)